docs/stale-subvolume.md

```diff
@@ -0,0 +1,38 @@
+# subvolume cleanup
+
+The subvolume command is used to clean the stale subvolume/images
+which has no parent-pvc attached to it.
+The command would list out all such subvolume/images which needs to be removed.
+This would consider all the cases where we can have stale subvolume
+and delete them without impacting other resources and attached volumes.
+
+Few cases are mentioned below:
+* Multiple filesystems in the ceph cluster
+* Multiple subvolume groups in the same filesystem
+* Subvolume with retain-snapshot feature (only snapshot exists on this subvolume)
+* Pending clones (PVC)
+
+The subvolume command will require the following sub commands:
+* `ls` : [ls](#ls) lists all the stale subvolume/images that can be removed
+* `clean` : [clean](#clean) stale subvolume/images after user's confirmation
+
+## ls
+
+```bash
+kubectl rook-ceph subvolume ls
+
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110004
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110005
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110006
+# csi-vol-427774b4-340b-11ed-8d66-0242ac110007
+
+```
+
+## clean
+
+```bash
```

Comment:
What is the behavior if the subvolume is not stale and still in use? Do we prevent removing it?

Reply:
yes, we will only list out the stale volume. I will elaborate more on what stale volume is. basically if a pvc is created with retain stratergy, after the deletion of pvc, the subvolumes stays. Those subvolumes are of no use and are stale. hence the user wants to delete it.